src/global/interfaces/link-button-properties.ts

```diff
@@ -76,6 +76,32 @@ export interface ButtonProperties<T = any> extends AccessibilityProperties {
   emitButtonClick: (() => void) | undefined;
 }
 
+/**
+ * A component that implements LinkButtonProperties should use this interface to set useful variables for render function.
+ */
+export interface LinkButtonRenderVariables {
+  /**
+   * The tag's name rendered by the component.
+   */
+  tagName: 'a' | 'button' | 'span';
+
+  /**
+   * The tag's attributes; can be set using getLinkButtonBaseAttributeList(...),
+   * getLinkAttributeList(...) or getButtonAttributeList(...) methods.
+   */
+  attributes: Record<string, string>;
+
+  /**
+   * The tag's CSS classes.
+   */
+  cssClass?: string;
```

Comment:
I'd prefer not to have cssClass in linkbuttonrendervariables, but maybe you can convince me ;-)

Reply:
I'll try 😄 : 
personally, I find `<sbb-card class='sbb-card sbb-card__link'/>` with the CSS rule for the `sbb-card__link` more readable then `<sbb-card class='sbb-card'/>` with rules like `button.sbb-card {...} a.sbb-card {...}`, but, as I said, is a personal preference

Reply:
what also would be possible is to use :host([href]) & {...} for link and :host(:not([href])) & {...} for button. But if this is more readable, I don't know ;)